README.md

```diff
@@ -34,6 +34,8 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+You will also need to add your own TRELLO_API_KEY and TRELLO_API_TOKEN to the .env file as this will not be stored between sessions.
```

Comment:
It would be good to also add the keys (but not the values) for these two to the `.env.template` file so that there's another hint there to remember to set them

Reply:
Have added the keys to the .env.template file